int32 marker_id
geometry_msgs/Transform marker_pose
float64 error